src/vs/workbench/parts/preferences/common/keybindingsEditorModel.ts

```diff
@@ -90,19 +90,72 @@ export class KeybindingsEditorModel extends EditorModel {
 
 	public fetch(searchValue: string, sortByPrecedence: boolean = false): IKeybindingItemEntry[] {
 		searchValue = searchValue.trim();
-		const quoteAtFirstChar = searchValue.charAt(0) === '"';
-		const quoteAtLastChar = searchValue.charAt(searchValue.length - 1) === '"';
+
+		if (this.isSourceFilterApplied(searchValue)) {
+			searchValue = this.getSourceFilterValue(searchValue);
+			searchValue = this.parseSearchValue(searchValue);
```

Comment:
Why do we need to `parseSearchValue`?

Reply:
This is added to handle the case with quotes and source filter is applied. --> `@source: "user"` / `@source: "default"`

Reply:
@shobhitchittora Lets make it simple and support only `@source:user` or `@source:default`. I do not think why we need to support quotes. This will make everything much simpler

Reply:
Yeah I probably over-complicated it, because I saw quotes were also supported. Will simplify it. 💪🏼